Remotion/Mixins/UnitTests/Core/CodeGeneration/AdHocCodeGenerator.cs

```diff
@@ -40,7 +40,11 @@ public AdHocCodeGenerator (string assemblyDirectory, string assemblyName = "AdHo
       ArgumentUtility.CheckNotNullOrEmpty ("assemblyName", assemblyName);
 
       _filename = assemblyName + ".dll";
+#if !NO_PEVERIFY
```

Comment:
```suggestion
#if NETFRAMEWORK && !NO_PEVERIFY
```

Reply:
is `!NO_PEVERIFY` sufficient or are both conditions needed?

Reply:
We need both conditions since a) this is only needed for PEVerify and b) still only works in Full Framework. And I didn't want to go and just kick PEVerify in the project setup for .net 5 as a means to get the project to compile. PEVerify yes/no should be a configuration feature, not a compilation feature (but that's a different subject)